### ISSUE TITLE
Keep track of prior model results using a results.json file

### ISSUE THREAD

**Issue by StephanAkkerman** [StephanAkkerman/chart-recognizer#5]
We track the results with prior parameters so we can see the changes in performance when changing parameters. Maybe we can also add a config.json for changing the settings of the model.

The parameters we should track are:
- Timm model name
- Batch size
- Datablock parameters
- Datasets used
- Number of images used
- Validation percentage
- Metrics results on val and test set
- Loss function
- Callbacks
- Epoch
- Learning rate
- Confusion matrix results (TP, TN, FP, FN)

- [x] Add config.json for changing parameters
- [x] Write model output + used parameters to results.json
